src/library/Uno.Cupertino/Styles/Controls/ComboBox.xaml

```diff
@@ -117,19 +116,13 @@
 
 						<!-- ContentPresenter-->
 						<ContentPresenter x:Name="ContentPresenter"
+										  Content="{TemplateBinding Content}"
+										  ContentTemplate="{TemplateBinding ContentTemplate}"
+										  ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}"
```

Comment:
Should we be setting the CupertinoFootnote FontSize/FontWeight directly  on the ContentPresenter then?

Reply:
the typography would follow this precendence:
this ContentPresenter -> ComboBoxItem style -> ComboBox style

if we were, it shouldve been on CB
and if the item style would be different (by default) than the CB, it should be on CBI
we should never hardcode that on the CP, as we cant override it without re-templating

Reply:
i dont know what is the font family/size/weight to be used here
so im hesitant to touch it